test/cpp/api/modules.cpp

```diff
@@ -454,6 +470,11 @@ TEST_F(ModulesTest, PrettyPrintLinear) {
       c10::str(Linear(3, 4)), "torch::nn::Linear(in=3, out=4, with_bias=true)");
 }
 
+TEST_F(ModulesTest, PrettyPrintBilinear) {
+  ASSERT_EQ(
+      c10::str(Bilinear(3, 2, 4)), "torch::nn::Bilinear(in1=3, in2=2, out=4, with_bias=true)");
```

Comment:
It would be fantastic to also test the `Bilinear(BilinearOptions(3, 2, 4).bias(false))` case. :) Thanks!